### ISSUE TITLE
"Debug: Error messageIncomplete" is displayed after STR

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6474]
## Preconditions:
1. Log in to FxA on device A. 
2. Login to FxA on device B.
3. On device A, go to Account Management Page -> Devices & App, and tap "Disconnect" for the current device and "Disconnect" for the current Web Session.
4. Close and re-launch the app on device A
5. Log in to FxA on device A. 
## Steps to reproduce
1. On device B, go to "github.com".
2. Open Page Action Menu.
3. Tap "Send to device"
4. Select device A from the list
### Expected behavior
- Tab should be sent across iOS devices
### Actual behavior
- "Debug: Error messageIncomplete=FirefoxAccountError.unauthorized: Remote server error: 401 110" notification is displayed on device A and the tab is not sent. 
### Device & build information
* Device: 
Device A -> iPhone 7 (13.4.1)
Device B -> iPhone Xr (13.3)
* Build version: 25.0 (17793)

**Comment by garvankeeley** [comment]
This should only be shown on a Beta build, can you check on a release build? Thanks

**Comment by garvankeeley** [comment]
When I am disconnecting my current device using another device, the current device is not disconnecting.

**Comment by garvankeeley** [comment]
Confirmed, I am seeing the notification content as 
```
<UNNotificationContent: 0x280bfdad0; title: Firefox Sync, subtitle: (null), body: DEBUG: Error messageIncomplete=FirefoxAccountError.unauthorized: No stored refresh token, summaryArgument: (null), summaryArgumentCount: 0, categoryIdentifier: , launchImageName: , threadIdentifier: , attachments: (
), badge: (null), sound: (null), realert: 0
```

**Comment by garvankeeley** [comment]
This is what I see in the debugger
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/5495083/80644318-7345a580-8a37-11ea-8d31-bdc67229f049.png">

The payload is 
```
"{\"version\":1,\"command\":\"fxaccounts:command_received\",\"data\":{\"command\":\"https://identity.mozilla.com/cmd/open-uri\",\"index\":11236102,\"sender\":\"9302650bc0eaad36a8f3d22348cb4ba5\",\"url\":\"https://api.accounts.firefox.com/v1/account/device/commands?index=11236102&limit=1\"}}"
```

**Comment by garvankeeley** [comment]
After restarting the app a second time it works for me, so at least it is self-healing.

**Comment by SimonBasca** [comment]
Verified fix on 25.0 (17803).
